tinywl/tinywl.c

```diff
@@ -90,6 +103,10 @@ struct tinywl_view {
 	struct wl_listener request_maximize;
 	struct wl_listener request_fullscreen;
 	int x, y;
+
+	struct wlr_addon addon;
+
+	struct tinywl_decoration_data deco_data;
```

Comment:
Let's have the corner radius and alpha be in the tinywl view struct, no need for a decoration data struct